src/Optical/OpticalSystem.jl

```diff
@@ -393,7 +387,7 @@ struct AxisymmetricOpticalSystem{T,C<:CSGOpticalSystem{T}} <: AbstractOpticalSys
                 )
             end
 
-            if firstelement && semidiameter !== NaN
+            if firstelement
                 systemsemidiameter = semidiameter
                 firstelement = false
             end
```

Comment:
This change will allow the Stop semidiameter to define the systemsemidiameter if it's the first element. This is different from the previous behaviour - is this OK?

Reply:
ask Joel Kollin. He could tell us if this makes sense optically.

Reply:
I asked the Cambridge team today and the general consensus was that individual semidiameters are important but not so much the system semidiameter. And, if anything, it'd be more useful to have this as the max value. I'll have a look around the code tomorrow to see if this is actually used anywhere currently.